app/schemas/federation_entities.json

```diff
@@ -0,0 +1,366 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "id": "https://diasporafoundation.org/schemas/federation_entities-1.0.0.json",
```

Comment:
As our schema seem to get stabilized (at least for this PR) we should decide upon publishing it at some real place. Also we should define versioning schema. I think using semver is okay for it. Maybe we should also make `-latest` symlink?